vcd/sample_vcd_test_config.json

```diff
@@ -29,6 +29,13 @@
     "//catalog": {
       "//name": "existingCatalogName",
       "//catalogItem": "existingOvaNameInCatalog"
+    },
+    "allocationModel": "ReservationPool",
```

Comment:
It looks an overkill on adding allocation model to test config, which leads to one more parameter... Can we test all models automatically?

Reply:
allocation model removed, test improved to cover all cases

Reply:
Thank you! @Didainius please review the change and close this conversation if it fits your ideas as well.

Reply:
Thanks! I like it -  we cover more and it's not that expensive in time:
```
=== RUN   TestAccVcdOrgVdcReservationPool
--- PASS: TestAccVcdOrgVdcReservationPool (18.30s)
=== RUN   TestAccVcdOrgVdcAllocationPool
--- PASS: TestAccVcdOrgVdcAllocationPool (17.10s)
=== RUN   TestAccVcdOrgVdcAllocationVApp
--- PASS: TestAccVcdOrgVdcAllocationVApp (20.04s)
PASS
Looking for resources to delete from test suite...
Catalog deletion skipped as user defined resource used
Catalog item deletion skipped as user defined resource is used or removed with catalog
ok      github.com/terraform-providers/terraform-provider-vcd/v2/vcd    57.514s
```